### PR TITLE
Configure CircleCI

### PR DESCRIPTION
**Pull request by mattsaxe17**

### CODE REVIEW

File: service/index.js [hunk 1]
```diff
@@ -24,6 +24,12 @@ module.exports = {
     return pool.query(q);
   },
 
+  deactivateUser(email) {
+    let q = `select up_user_deactivate('${email}')`;
```
Comment (aleksandar-cakic) [inline service/index.js:28]: I'm curious about the naming convention, why is it up_user?

File: test/routes.test.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    const response = await request
+      .post('/deactivate')
+      .send({
+        email: 'joemama@email.com',
+      })
+      .set('x-access-token', token);
+
+    expect(response.status).toBe(200);
+  });
+});
```
Comment (aleksandar-cakic) [inline test/routes.test.js:103]: Awesome tests :)

**Review by aleksandar-cakic — APPROVED** [review]

File: service/index.js (hunk 1 above)
Comment (mattsaxe17) [reply service/index.js:28]: this is how function and stored procedures are often named in the industry. up stands for 'user procedure'. Where i work we use up_NAME_HERE for stored procedures and fn_NAME_HERE for functions, but thats sql server. postgres is a bit different, and functions in postgres are like stored procedures in sql server. I looked it up and the convention is still to use up_... i believe. hope this helps!

File: test/routes.test.js (hunk 2 above)
Comment (mattsaxe17) [reply test/routes.test.js:103]: thanks!!

**Pull request merged**